src/presentation/pages/Categories/Creatures/CreaturesPage.tsx

```diff
@@ -0,0 +1,16 @@
+import { TopBarLayout } from 'infrastructure/components/Layout/TopBarLayout';
+import { MainContent } from './Main/Main';
+
+const CreaturesPage = () => {
+  return (
+    <TopBarLayout
+      pageComponent={
+        <>
+          <MainContent />
+        </>
```

Comment:
The empty fragment is not needed
```suggestion
          <MainContent />
```